src/components/Picklist/readme.md

```diff
@@ -247,3 +247,37 @@ class PicklistExample extends React.Component {
 
 <PicklistExample />;
 ```
+##### Picklist error
+
+```js
+import React from 'react';
+import { Picklist, PicklistOption } from 'react-rainbow-components';
+
+const containerStyles = {
+    width: '200px',
+};
+
+<div className="rainbow-m-bottom_xx-large rainbow-p-bottom_xx-large">
+    <GlobalHeader
+        src="images/user/user2.jpg"
+        className="rainbow-p-bottom_xx-large rainbow-m-bottom_xx-large"
+    >
+        <div className="rainbow-flex rainbow-align_right">
+            <Picklist
+                id="picklist-1"
+                style={containerStyles}
+                onChange={value => setState({ value })}
+                value={state.value}
+                error= {!state.value ? "This Field is Required" : null}
+                label="Select Building"
+                hideLabel
+            >
```

Comment:
Change the example, delete the GlobalHeader and make it more similar to the example of the Select component. 

Do not change the state of the error, leave it fixed, the idea is to show the error.

Do not put id and if you put it the id has to be "picklist-11" because the id of the examples are the sequence of the odd numbers, that is connected with the integration-tests.

Reply:
👍